DataFormats/HcalCalibObjects/src/HFRecalibration.cc

```diff
@@ -35,7 +35,7 @@ double HFRecalibration::getCorr(int ieta, int depth, double lumi) {
   switch (depth) {
     case 1:
       reCalFactor = (1 + HFParsAB[0][0][ieta] * sqrt(lumi) + HFParsAB[0][1][ieta] * lumi);
-
+      [[fallthrough]];
```

Comment:
I suspect this wasn't originally supposed to fall through: @abdoulline @igv4321 ?
(maybe a few others in this PR...)

Reply:
@perrotta 
I suppose there had to be "break" originally (apparently forgotten by author), as HF depths are treated separately (method parameter list has "depth" parameter). 
So fall though  doesn't seem to be natural here...